x-pack/test/api_integration/apis/apm/feature_controls.ts

```diff
@@ -20,122 +20,184 @@ export default function featureControlsTests({ getService }: FtrProviderContext)
   const expect404 = (result: any) => {
     expect(result.error).to.be(undefined);
     expect(result.response).not.to.be(undefined);
-    expect(result.response).to.have.property('statusCode', 404);
+    expect(result.response.statusCode).to.be(404);
```

Comment:
This makes the output much cleaner. Instead of outputting `Expected {...hugeObject} to have property statusCode 404` it will now output `Expected 500 to be 404`